src/components/custom-procedures/custom-procedures.jsx

```diff
@@ -14,6 +14,7 @@ const CustomProcedures = props => (
     <Modal
         className={styles.modalContent}
         contentLabel="Make a Block"
+        onKeyPress={props.onKeyPress}
```

Comment:
You will need to pass this in from the custom procedure container, right now nothing is being passed for `onKeyPress`. Should be added around line 110 of src/containers/custom-procedures.jsx

Reply:
Should work now